tools/src/main/java/org/apache/kafka/tools/StreamsResetter.java

```diff
@@ -93,7 +93,7 @@ public class StreamsResetter {
     private static final String USAGE = "This tool helps to quickly reset an application in order to reprocess "
             + "its data from scratch.\n"
             + "* This tool resets offsets of input topics to the earliest available offset (by default), or to a specific defined position"
-            + " and it skips to the end of intermediate topics (topics that are input and output topics, e.g., used by deprecated through() method).\n"
+            + " and it skips to the end of intermediate topics (topics that are input and output topics.\n"
```

Comment:
> and it skips to the end of intermediate topics (topics that are input and output topics.

This part can be removed too -- w/o `through()` there is no such thing as "intermediate topics" and longer.

Reply:
Good Catch, I have just patched that in the latest commit.